### PR TITLE
PPTP-1641 : Fixed Up Partnership Task-list

### PR DESCRIPTION
**Pull request by RyanKellyHMRC**
#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [x] User Acceptance Tests (UAT) were run locally and have passed.

### CODE REVIEW

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ReviewRegistrationController.scala [hunk 1]
```diff
@@ -69,40 +62,16 @@ class ReviewRegistrationController @Inject() (
...
-          // The call to check that the registration is in a suitable state before this means that
-          // exhaustive matching is not needed
-          (request.registration.organisationDetails.organisationType: @unchecked) match {
-            case Some(UK_COMPANY) | Some(REGISTERED_SOCIETY) | Some(OVERSEAS_COMPANY_UK_BRANCH) =>
-              incorpEntityReview()
-            case Some(SOLE_TRADER) => soleTraderReview()
-            case Some(PARTNERSHIP) => partnershipReview()
-          }
+          Ok(
+            reviewRegistrationPage(registration = request.registration,
```
Comment (tonymccrae-ee) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ReviewRegistrationController.scala:66]: Yep; stick this problem in a function; this is good.

File: app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/Registration.scala [hunk 2]
```diff
@@ -60,23 +60,41 @@ case class Registration(
...
-          isPartnership && isNominatedPartnerDetailsComplete,
+          isPartnershipWithPartnerCollection && isNominatedPartnerDetailsComplete && isOtherPartnersDetailsComplete,
           isRegistrationComplete
     ).count(isComplete => isComplete)
 
   def isGroup: Boolean = registrationType.contains(RegType.GROUP)
 
   def isPartnership: Boolean = organisationDetails.organisationType.contains(OrgType.PARTNERSHIP)
 
+  def isPartnershipWithPartnerCollection: Boolean =
```
Comment (tonymccrae-ee) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/Registration.scala:73]: Can some of these be marked as private?

File: build.sbt [hunk 3]
```diff
@@ -49,7 +49,7 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     ".*(BuildInfo|Routes|Options).*",
     "logger.*\\(.*\\)",
   ).mkString(";"),
-  coverageMinimum := 95.1,
+  coverageMinimum := 95.0,
```
Comment (tonymccrae-ee) [inline build.sbt:52]: :+1:

**Review by tonymccrae-ee — APPROVED** [review]

File: app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/Registration.scala (hunk 2 above)
Comment (RyanKellyHMRC) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/Registration.scala:73]: Yup, a couple can and have.

**Pull request merged**